packages/shared/src/loadClerkJsScript.ts

```diff
@@ -59,6 +59,40 @@ function isClerkProperlyLoaded(): boolean {
   return typeof clerk === 'object' && typeof clerk.load === 'function';
 }
 
+/**
+ * Checks if an existing script has a request error using Performance API.
+ *
+ * @param scriptUrl - The URL of the script to check.
+ * @returns True if the script has failed to load due to a network/HTTP error.
+ */
+function hasScriptRequestError(scriptUrl: string): boolean {
```

Comment:
This is a slightly cursed way trying to check for an existing script load error without the ability to add an `onError` callback to the script. `window.performance.getEntries()` should contain the network request to fetch clerk-js, and so we can better infer what the status is without an arbitrary timeout.

Reply:
I like this!